google/resource-snippets/pubsub-v1/alternatives/krm/pubsub.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: pubsub.cnrm.cloud.google.com/v1beta1
+kind: PubSubTopic
+metadata:
+  name: {$PUBSUB?}-topic. # {"$ref":"#/definitions/io.k8s.cli.substitutions.topic-subs"}
```

Comment:
`{$PUBSUB?}` should be `${PUBSUB?}` I believe. I checked out this GitHub and I can't do `kpt cfg set . PUBSUB foo` because of an error. Changing that makes the `kpt cfg set` command work.

Reply:
Also the period after `topic` should be removed.